conf/modules.config

```diff
@@ -741,6 +741,22 @@ process {
         ]
     }
 
+    withName: MAG_TO_SAMPLESHEET {
+        publishDir = [
+            path: { "${params.outdir}/samplesheet" },
+            enabled: false
+        ]
+    }
+
+    withName: 'MAG_MERGE_SAMPLESHEET' {
+        ext.prefix = "${params.nf_core_pipeline}"
+        publishDir = [
+            path: { "${params.outdir}/samplesheet" },
```

Comment:
`samplesheet` seems a bit generic, we spit out a 'validated' samplesheet in `piepline_info`. What about `downstream_samplesheet` or something... I know that's a bit ugly but I'm open to suggestions